packages/amplify-util-mock/src/__e2e__/subscriptions-with-auth.e2e.test.ts

```diff
@@ -247,41 +247,6 @@ test('Test that only authorized members are allowed to view subscriptions', asyn
   return subscriptionPromise;
 });
 
-test('Test that a user not in the group is not allowed to view the subscription', async () => {
```

Comment:
This test should not throw but there should not be any event received in the subscription. Since the async callback never gets invoked, the promise remains unresolved causing the test to timeout and fail ungracefully. Verified this behavior from console.